Gemfile

```diff
@@ -47,8 +47,10 @@ group :development, :test do
     when 'sqlite3'
       if orm == 'active_record' && orm_version < '5.2'
```

Comment:
We don't need this anymore since Mobility doesn't test against anything earlier than 6.1. Can you remove this conditional branch and default to sqlite ~> 1.5.0 ?

Reply:
done!